.evg.yml

```diff
@@ -7,10 +7,11 @@ functions:
     - command: shell.exec
       params:
         script: |
-          git clone https://github.com/realm/SwiftLint
-          cd SwiftLint
-          git submodule update --init --recursive
-          make install
+          mkdir -p SwiftLint
+          pushd SwiftLint
+          curl -o swiftlint.zip -L https://github.com/realm/SwiftLint/releases/download/0.29.1/portable_swiftlint.zip
+          unzip swiftlint.zip
+          popd
```

Comment:
generally fine but what's the behavior if curl or unzip fails? does the script abort or continue?

Reply:
I'll add a `set -e` so the script aborts